docs/UserGuide.md

```diff
@@ -192,8 +192,8 @@ the constraints of each parameter when used in a command.
 {: .alert .alert-warning}
 > :exclamation: **Caution:** 
 >
-> Ensure that all parameters adhere to the constraints mentioned below. 
-> Otherwise, the command will not be executed, and an error message will be displayed.
+> * Ensure that all parameters adhere to the constraints mentioned below. Otherwise, the command will not be executed, and an error message will be displayed.
```

Comment:
```md
* Ensure that all parameters adhere to the constraints mentioned below. Otherwise, the command will not be executed, and an error message will be displayed.
  * This only applies to commands that accept preambles / parameters
  * Commands like `list` and `help` that do not accept preambles / parameters will simply ignore anything added after the command word and executes. 
```

@hweinian